sdk/storage/azure-storage-blob/tests/test_container.py

```diff
@@ -1166,9 +1166,47 @@ def test_delete_blobs_simple(self, resource_group, location, storage_account, st
         assert response[1].status_code == 202
         assert response[2].status_code == 202
 
+    @pytest.mark.live_test_only
     @pytest.mark.skipif(sys.version_info < (3, 0), reason="Batch not supported on Python 2.7")
-    @GlobalResourceGroupPreparer()
-    @StorageAccountPreparer(random_name_enabled=True, location="canadacentral", name_prefix='storagename')
```

Comment:
it seems tags only works in canadacentral (at least previously), are we sure removing this will not break live test?